core/src/main/java/com/nerds/gamejam/util/TextureLoader.java

```diff
@@ -0,0 +1,14 @@
+package com.nerds.gamejam.util;
+
+import com.badlogic.gdx.graphics.g2d.Animation;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.nerds.gamejam.ecs.component.AnimationComponent;
+import com.nerds.gamejam.ecs.component.TextureReferenceComponent.TextureReference;
+
+public interface TextureLoader {
```

Comment:
What other implementations do you have planned for this?

Reply:
Probably nothing, tbh this started out as a lambda so I needed the interface